ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -0,0 +1,193 @@
+                selectedAccount: selectFromAccount.selectedAccount
+                defaultCurrency: walletModel.defaultCurrency
+                getFiatValue: walletModel.getFiatValue
+                getCryptoValue: walletModel.getCryptoValue
+                width: stack.width
+                reset: function() {
+                    selectedAccount = Qt.binding(function() { return selectFromAccount.selectedAccount })
+                }
+            }
+        }
+        TransactionFormGroup {
+            id: group3
+            headerText: qsTr("Transaction preview")
+            footerText: root.finalButtonLabel
+
+            TransactionPreview {
+                id: pvwTransaction
+                width: stack.width
+                fromAccount: selectFromAccount.selectedAccount
+                toAccount: selectRecipient.selectedRecipient
+                asset: txtAmount.selectedAsset
+                amount: { "value": txtAmount.selectedAmount, "fiatValue": txtAmount.selectedFiatAmount }
+                currency: walletModel.defaultCurrency
+                reset: function() {
+                    fromAccount = Qt.binding(function() { return selectFromAccount.selectedAccount })
+                    toAccount = Qt.binding(function() { return selectRecipient.selectedRecipient })
+                    asset = Qt.binding(function() { return txtAmount.selectedAsset })
+                    amount = Qt.binding(function() { return { "value": txtAmount.selectedAmount, "fiatValue": txtAmount.selectedFiatAmount } })
+                }
+            }
+
+            SVGImage {
+                width: 16
+                height: 16
+                source: "../../../../img/warning.svg"
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.bottom: warningText.top
+                anchors.bottomMargin: 4
+            }
+
+            StyledText {
+                id: warningText
+                text: qsTr("You need to request the recipient’s address first.\nAssets won’t be sent yet.")
+                color: Style.current.danger
+                horizontalAlignment: Text.AlignHCenter
+                wrapMode: Text.WordWrap
+                anchors.right: parent.right
+                anchors.left: parent.left
+                anchors.bottom: parent.bottom
+
+            }
+        }
+    }
+
+    footer: Item {
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+        StyledButton {
+            id: btnBack
+            anchors.left: parent.left
+            width: 44
+            height: 44
+            visible: !stack.isFirstGroup
+            label: ""
+            background: Rectangle {
+                anchors.fill: parent
+                border.width: 0
+                radius: width / 2
+                color: btnBack.disabled ? Style.current.grey :
+                        btnBack.hovered ? Qt.darker(btnBack.btnColor, 1.1) : btnBack.btnColor
```

Comment:
At what point is this button disabled? Maybe this isn't needed? Also, do we have a dark mode version of `grey` (IOW, maybe we could create/use `Style.current.disabled` instead)?